Common/Sources/Helper/ThemeManager.swift

```diff
@@ -59,7 +59,7 @@ public class ThemeManager: NSObject {
             navBarAppearance.largeTitleTextAttributes =
                 [
                     NSAttributedString.Key.foregroundColor: Theme.NavigationBar.largeTitleColor,
-                    NSAttributedString.Key.font: Theme.NavigationBar.largeTitleFont
+                    NSAttributedString.Key.font: UIFont.boldSystemFont(ofSize: 20.0)
```

Comment:
I thought the idea was to use sizeToFit() instead of using smaller font size

Reply:
Actually sizeToFit() can use navigationbar titleView, which will come in center. So i have choosed this smaller font size.